hooks/useWeb3.ts

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from 'react'
+import { useDisconnect } from 'wagmi'
+
+import { ModalCategory } from 'app/states/modal'
+import { useModal } from './useModal'
+import { useSettings } from './useSettings'
+import { useTx } from './useTx'
+
+export function useWeb3() {
```

Comment:
Non-descriptive hook name. Not a big deal if you're working alone, but a huge deal if someone joins. I suggest something like `useConnectionActions` or something (I'm bad at naming things).

Reply:
Tbh, I did think `useWeb3` was pretty okay. 😂 hahaha
But I like what you suggested more. How about `useConnectActions` then?

Reply:
LGTM